packages/sfui/shared/scrollable/scrollable.ts

```diff
@@ -183,36 +189,34 @@ export default class SFScrollable {
     const container = this.container;
     if (!container) return;
 
-    this.refresh((data) => this.options.onScroll?.(data));
+    this.refresh(this.options.onScroll);
   }
 
   private onScrollHandler() {
-    this.refresh((data) => this.options.onScroll?.(data));
+    this.refresh(this.options.onScroll);
   }
 
-  private calculate(): SfScrollableCallbackData {
-    const { container, options } = this;
-
-    function hasNext() {
-      if (options.vertical) {
-        return container.scrollHeight > Math.ceil(container.scrollTop) + container.clientHeight;
-      }
-      return container.scrollWidth > Math.ceil(container.scrollLeft) + container.clientWidth;
+  private get hasNext() {
+    if (this.options.direction === SfScrollableDirection.vertical) {
+      return this.container.scrollHeight > this.container.scrollTop + this.container.clientHeight;
     }
+    return this.container.scrollWidth > this.container.scrollLeft + this.container.clientWidth;
+  }
```

Comment:
Device:
Chrome `Version 112.0.5615.49 (Official Build) (arm64)`
IOS `13.3.1`

I was testing in development build, on preview page `http://localhost:3002/examples/SfScrollable`

Reply:
@Razz21 I just created [ticket](https://vsf.atlassian.net/browse/SFUI2-1011), I need to spend more time for looking into it, since your team rely on this PR, and thing you said not working is in hook, it will be fixed in package, therefor will not  require additional work on your side, only on our. I would not like to block your team atm